### PR TITLE
Add Windows IIS to docs

### PR DESCRIPTION
**Pull request by jsreynolds**
Added Windows IIS proxy instructions and Powershell import of previous or new packages to documentation.

Docs only

### CODE REVIEW

File: docs/tools/powershell-import.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Powershell import
+
+For moving an existing repo or adding NuGet packages in bulk, this may help. It simply creates a .bat file which can be executed for import.
+
+```powershell
+Get-ChildItem -Path "C:\Temp\NewPackages" -Recurse | Where-Object {$_.Extension -eq ".nupkg"} | Select-Object FullName | ForEach-Object {
```
Comment (loic-sharma) [inline docs/tools/powershell-import.md]: I like the idea of guiding users on how to bulk import packages, but this is already possible using the following command:

```
dotnet nuget push *.nupkg -s http://myserver:5000/v3/index.json -k {apikey}
```

Let's remove this document for now, and I'll add this command into the getting started steps

File: docs/windows/iis-proxy.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Using IIS Manager, create a new web site:
+
+- Choose your site name and physical path
+- Choose BaGetAppPool as the application pool
+- In the Binding area, enter the default BaGet port of 5000
+
+## BaGet Folder permissions
+
+In order for the app to create the appropriate NuGet package folder, as well as create various files (SqLite database for example) you **may** need to give special rights to the top level BaGet folder.
```
Comment (loic-sharma) [inline docs/windows/iis-proxy.md]: `SqLite` should be `SQLite`

File: docs/windows/iis-proxy.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+In order for the app to create the appropriate NuGet package folder, as well as create various files (SqLite database for example) you **may** need to give special rights to the top level BaGet folder. 
+
+Read under, "Application Pools" and, "Application Pool Identity" [here](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/?view=aspnetcore-2.2). Basically the identity used in the app pool isn't a real user account and doesn't show up in the Windows User Management Console.
+
+## Alternative NuGet package Storage Paths
+
+Note that Virtual Directories will not work with IIS and Kestrel. Read more about that [here](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/?view=aspnetcore-2.2)
+
+Ensure that the configuration storage Path uses approprite forward slashes in the settings such as:
```
Comment (loic-sharma) [inline docs/windows/iis-proxy.md]: `approprite` should be `appropriate`

**Comment by loic-sharma** [comment]
Looks great! I've left some comments, I'll merge this in once you've addressed them. Thank you :)

**Comment by jsreynolds** [comment]
Revisions finished - thanks for the feedback.   :-)

**Comment by loic-sharma** [comment]
Sweet, your changes are now live: https://loic-sharma.github.io/BaGet/windows/iis-proxy/

Thanks!

**Pull request merged**